cli/src/commands/unsquash.rs

```diff
@@ -56,6 +57,14 @@ pub(crate) fn cmd_unsquash(
     command: &CommandHelper,
     args: &UnsquashArgs,
 ) -> Result<(), CommandError> {
+    writeln!(
+        ui.warning_default(),
+        "`jj unsquash` is deprecated; use `jj squash` or `jj diffedit` instead"
```

Comment:
Nit: shouldn't that be `jj diffedit --restore-descendants`?

Reply:
I think `diffedit --restore-descendants` is the better alternative for `jj unsquash`'s behavior, but let's wait for the opinions of others.

Reply:
Also, I got convinced for diffedit --restore-descendants, I even put it before squash in the deprecation message, it makes sense.